ovos_padatious/padaos.py

```
@@ -93,32 +145,56 @@
+        Args:
+            line (str): The line to compile into a regex.
+            intent_name (str): The name of the intent.
+
+        Returns:
+            Optional[re.Pattern]: The compiled regex pattern or None if an error occurs.
+        """
         try:
-            return re.compile(self._create_intent_pattern(line, intent_name),
-                              re.IGNORECASE)
+            return re.compile(self._create_intent_pattern(line, intent_name), re.IGNORECASE)
         except Exception as e:
```

Ruff

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)